webui/app.py

```diff
@@ -113,27 +118,42 @@ def stammdaten():
         zustand = request.form.get('zustand')
 
         if len(besucher) == 0:
-            message = 'Neuer Besucher mit ID {} hinzugef&uuml;gt.'.format(besucher_id)
-            print('Adding new user: {}'.format(besucher_id))
-            cursor.execute(new_user_stmt, (besucher_id,
-                                           name, adresse1, plz, adresse2,
-                                           telefon,email,status,coronawarn))
+            # check district
+            cursor.execute(get_district_stmt, (plz,))
+            landkreis = cursor.fetchall()
+            if len(landkreis) == 0:
+                lvl = 'warning'
+                print('PLZ kann keinem Landkreis zugeordnet werden.')
```

Comment:
This should also be the `message` right? This code path will not add the user entry.

Reply:
Alternatively, allow such entries. Leave the `lvl` at `info` and only set the message.

Reply:
Agreed on both.